client/packages/system/src/Patient/Components/PatientSearchInput/PatientSearchInput.tsx

```diff
@@ -53,11 +67,12 @@ export const PatientSearchInput: FC<NameSearchInputProps> = ({
           setInput(value);
           search(value);
         },
-        // reset input value to previous selected patient if user clicks away without selecting a patient
+        // reset input value to previous selected patient if user clicks away
+        // without selecting a patient
         onBlur: () => setInput(value?.name ?? ''),
       }}
-      filterOptions={filterByNameAndCode}
-      sx={{ minWidth: width }}
+      filterOptions={options => options}
```

Comment:
It seems annoying to have to keep this prop in, but it seems necessary as the default behaviour doesn't work. But we don't need the `filterByNameAndCode` method, since the `useSearchPatient` is already doing that for us.